data/gen.go

```diff
@@ -577,20 +589,18 @@ func main() {
 
 		stat_id := parseInt(record[1]) - 1
 		stat_value := parseInt(record[2])
-
-		base_stat_array[dex_num][stat_id] = stat_value
+		ev := parseInt(record[3])
+		baseStats[dex_num].stats[stat_id] = stat_value
+		baseStats[dex_num].ev[stat_id] = ev
 	}
 
-	for dex_num, stats := range base_stat_array {
-
+	for dex_num, v := range baseStats {
```

Comment:
It looks like its swapping around the order of the items in `pokemonBaseStats`. The iteration order of a map is not guaranteed by the go compiler.

Change this so that it iterates according to the length of the map, so something like
```go
for i := 1; i < len(baseStats); i++ {
```